tools/caffe_translator/README.md

```diff
@@ -0,0 +1,78 @@
+# Caffe Translator
+Caffe Translator is a source code translator that translates Caffe code into MXNet Python code. Note that this is different from the Caffe to MXNet model converted which is available [here](https://github.com/apache/incubator-mxnet/tree/master/tools/caffe_converter).
+
```

Comment:
You mention the "what" then move to the "how", but I think you need a "why". What's the use case here. I'm still not sure why you would do this.

Reply:
Will it help if I change this to:

"Caffe Translator is a migration tool that helps developers migrate their existing Caffe code to MXNet and continue further development using MXNet. Note that this is different from the Caffe to MXNet model converter which is available [here](https://github.com/apache/incubator-mxnet/tree/master/tools/caffe_converter)."

I think mentioning that it is a migration tool to migrate code covers the "Why would you use this tool?" question.

Reply:
Made the change suggested above.